File: python_programming/practices/conditionals.py
#Cp 7 conditionals notes

#homework_done=input("is your homework done:")

#if homework_done=="no":
    #print("then go do your homework!")

#else:
   # print("yes you can go!")

#insie an if statement you ut a condition
#if statement checks wheather a condition is ture if its fure it runs the lock o code underneath
#boolean statements are expressions tha result in rue or false
#statements run when the if condition is not true
#you use an if elif else statement

#5 < 10	
#>8 > 3	
#>=10 >= 6	True, 
#==9 != 5to	4 == 4	True
#!=	Not equal to

grade = 100

if grade > 89:
    print(f"you have {grade}% that is an A!")
    if grade>79:
        print(f"you have {grade}% that is a B!")
    else:
        print(f"you have a {grade}% that is a F!")

        #logical operators and, or not
        
        name= input("what is your name:")

        if name=="ms laRose":
            print("you are the teacher")
        elif name=="tia" or name == "ashley":
         print("you are the TA")
        else:
            print(f"hello {name},you are a student!")

File: C_programming/conditionals.c
//CP 7th conditional notes 

#include <stdio.h>
#include <string.h>
/* logical operators
&& and 
||or 
! not
*/
int main (void){
    int grade;
    char name[50];
    printf("what is your grade percent:");
    scanf("%d",grade);
    printf("what is your name;");
    scanf("%s", &name);
    printf("%d", strcmp (name, "Ms.LaRose"));
    if(name == "Ms.Larose"){
        printf("you dont get a grade!\n");
    }
    
    else if(grade >= 90){
        printf("you have a A!!\n");
    }else if(grade >= 80){
    printf("you have a B!\n");
    }else if(grade >= 70){
    } else if (grade >= 70){
    } else{
        printf("you are failing: (\n");
    }
    return 0;
}







//What puts something inside of the “if” statement?
//curley brackts

//How are conditions written in C?
//if
//else if
//else

//How do we write elif conditions in C?
//else if

//When do else conditions run?
//when none of the other conditions come out ture 

//What are the 3 logical operators in C?
// < > >= <=